src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -4,23 +4,21 @@ $white: #ffffff;
 $room-info-bg: #005ee5;
 $border-radius: 28px;
 
-.party-map-content-container {
+.party-map-page-container {
```

Comment:
What was the purpose/reasoning behind this change?

It seems to be used within the `Map` component, which isn't a 'page' (which has a specific defined meaning in the organisms/molecules/etc file structure)

Reply:
https://github.com/sparkletown/sparkle/pull/1074/files#r556286544 since it not only has party-map content, but chat as well, I have decided that this naming would be more meaningful

Reply:
I think the pattern we have been using elsewhere is to relate it to the component, so it would be `.party-map-component`.

Then making use of css selectors/nesting, the content can just be `.party-map-component .content` (though using scss' nesting rather than writing out the full selector manually), and the sidebar can be a similarly nested `.party-map-component .sidebar` (but again, using scss nesting)